.gitignore

```diff
@@ -91,7 +91,7 @@ out
 
 # Nuxt.js build / generate output
 .nuxt
-dist
+#dist # Required by GitHub Action.
```

Comment:
Yes, if you look at the top of the file (cut off here in the preview), you'll see that it's a copy from GitHub's standard. I've commented out the line that doesn't work for us rather than deleting it, to make updates in the future easier.

The dist folder is needed, by the way, as GitHub Actions require a copy of the compiled code within the repo to be picked up by GitHub. We can't unfortunately just compile at runtime.